pty/pty_other.go

```diff
@@ -95,13 +100,21 @@ func (p *otherPty) Input() ReadWriter {
 	}
 }
 
+func (p *otherPty) InputWriter() io.Writer {
+	return p.pty
+}
+
 func (p *otherPty) Output() ReadWriter {
 	return ReadWriter{
-		Reader: p.pty,
+		Reader: &ptmReader{p.pty},
 		Writer: p.tty,
 	}
 }
 
+func (p *otherPty) OutputReader() io.Reader {
+	return &ptmReader{p.pty}
```

Comment:
I don't think that applies for copying _from_ an `*os.File`.  `io.Copy()` looks for `WriteTo()` on the source, [which `*os.File` does not implement](https://pkg.go.dev/os#File).

It does apply when you are copying _to_ an `*os.File`, since it implements `ReadFrom()` --- `InputWriter()` still returns the `*os.File` directly, so we can take advantage of that optimization.